src/pages/vendors/components/Contacts.tsx

```diff
@@ -0,0 +1,69 @@
+/**
+ * Invoice Ninja (https://invoiceninja.com).
+ *
+ * @copyright Copyright (c) 2021. Invoice Ninja LLC (https://invoiceninja.com)
+ *
+ * @copyright Copyright (c) 2021. Invoice Ninja LLC (https://invoiceninja.com)
+ *
+ * @license https://www.elastic.co/licensing/elastic-license
+ */
+
+import { Card, Element } from '@invoiceninja/cards';
+import { random } from '../../../common/helpers/id';
+
+import { Plus } from 'react-feather';
+import { useTranslation } from 'react-i18next';
+import { Contact } from './Contact';
+
+type Props = { data?: any; formik?: any };
+
+export function Contacts(props: Props) {
+  const [t] = useTranslation();
+  return (
+    <Card title={t('contacts')}>
+      <Element leftSide={t('contacts')}>
+        {props.data &&
+          props.data.map((contact: any, index: any) => {
+            return (
+              <Contact
+                data={contact}
+                key={index}
+                index={index}
+                formik={props.formik}
+              />
+            );
+          })}
+        <button
+          className="w-full py-2 inline-flex justify-center items-center space-x-2"
+          onClick={(event: any) => {
```

Comment:
Please extract this into a separate method, looking at it this way it looks really bad. Try to separate UI from the functionality as much as possible.